### DISCUSSION TITLE
Start Arena Season 2 with the release of Tempest Keep

### DISCUSSION THREAD

**Discussion by TotallyNotMyAccount** [chromiecraft/chromiecraft#6585] [Suggestions]
I’m sure we are all well aware that arena has many problems currently that drive down participation. The release of SSC has worsened this, to the point where 5s isn’t even happening anymore and 3s only on Tuesdays. A big part of the reason for that is that the season is old, and the available PvP gear is getting outpaced by raid gear. Nowhere is that more obvious then when we look at weapons. Plenty of people are already running around with the 3rd tier of the blacksmithing weapons and some healers even have access to the Vashj mace that is a whopping 18 item levels ahead of the pvp equivalent! This also means that pvp gear isn’t a factor for mid-tier guilds when gearing up, because the gear is so outdated it just can’t compete. But in TBC proper people wanting pvp gear for pve was one of the main drivers behind arena participation, since it was very competitive back then. 
When trying to revitalize arena, in my mind there’s no better way than to start season 2 asap, so people can go for the new gear. The additional resilience would also help decrease the insane burstiness of current arena, which is major factor in keeping people out of it as well. This would even have positive knock-on effects for battlegrounds, since the arena points provided by CTA would also go up in demand.
As a last point, the season is already way too long, and according to the CC website it’s supposed to last nearly another half a year. Don’t do that to us, release S2 already.

**Comment by TotallyNotMyAccount** [comment]
Oh, and to add to this: All problems mentioned here would be compounded by TK gear becoming available without S2 gear to counter-balance it.

**Comment by UndoUreche** [comment]
Yes. Just yes.

**Comment by sassouiabd** [comment]


**Reply by UndoUreche** [reply]
I'm sorry but this can simply be rephrased as "I don't believe you". There is no argument or reasoning that I can see in these points. The first is a jab at the person, the second is expressing lack of belief, the third is another jab at the person. This is a text dump of precisely nothing.

**Reply by UndoUreche** [reply]
Just elaborate, please

**Reply by TotallyNotMyAccount** [reply]
1- Please elaborate what you mean with this. As it stands this is ad hominem without any substance. Fact is that back during original TBC arena gear was used in raids a ton by mid tier guilds and below.

3- People who played since the start of CC arena are nearing BiS state or are even already there if they are high ranked. Being able to mix in broken T5 items into your PvP gear instead of spending arena points for those slots only speeds up the process. As mentioned in my original post, weapons are a prime example of this. People who are BiS or close to it lose motivation, you can see that with raid attendance in most guilds as well right now. If you don’t think that powerful new items up for grabs motivate people to play more then I would say that it is instead you who doesn’t fully comprehend the game we are playing here.

Your tone is also incredibly hostile and pejorative, and frankly put, I can't see anything in my original post that warrants such a reaction in any way. Check yourself.

2-
S1
```
Gladiator's Ornamented Spaulders
1160 Armor
+24 Intellect
+30 Stamina
Equip: Improves resilience rating by 20.
Equip: Improves critical strike rating by 23.
Equip: Increases spell power by 19.
```

S2
```
Merciless Gladiator's Ornamented Spaulders
1278 Armor
+24 Intellect
+36 Stamina
Equip: Improves resilience rating by 23.
Equip: Improves critical strike rating by 25.
Equip: Increases spell power by 23.
```

Sure looks like one of these items has more resilience than the other. You are now going to step in and tell me that a 15% increase isn't "significant" or that this or that item doesn't get as much of a boost. The fact is that S2 gear is a hell of a lot more competitive with T5 gear than S1 is and acquiring it would give people more of an incentive to queue than currently for sure. 

If you have ideas for revitalizing arena beyond introducing S2 I'd be happy to hear them. However, your criticism here is far from constructive and aggressive to boot. But what I really want to know, what's the harm even if it doesn't help? Spawning a few NPC vendors and the gear on them isn't going to eat into dev resources in any significant way. PvE gear is still far better for PvE. PvP guys get something to do that hopefully drives up activity.

And to add another point:
Season 1 is already half a year old, according to the original plan S2 is going to come out in roughly 4-5 months. Lets call that a year of S1 or nearly. Observation shows that brackets usually last around 3 months, if we extrapolate from that then according to current planning S2 and S3 are going to last 6 months each, while S4 is going to last a measly 3 months. I don't think I need to elaborate much on why that isn't a great idea. Or are we supposed to play S4 during the 71-74 bracket with completely changed balance and DKs in there to mess it up further? And wotlk dungeon gear in the mix as well? But even if we did something crazy like that S1 would still be 2x longer than any other season, and if we go by the original plan would be 4x longer than S4!

**Reply by vvranceanu** [reply]
the mesiah has spoken. thau shall stop arguing

**Comment by caffidev** [comment]
Yes.

**Comment by balleny** [comment]
PvP and Arena is already in a very bad state given multiple factors like wotlk talents but tbc stats, serversize, bugs, etc.
Additional PvE gear increasing the missing balance is only the icing on the cake.

Effectively, moving to the next arena season, i would see the following results happen:
- Initial spike in battlegrounds as people want to get the new offpieces - on retail, a season switch would include a reset in arena points, honor and token - without reset, the demand for battleground would be less because of already farmed honor and marks.
- Initial spike in arena to upgrade items. likely some regular 5s for minimal points, and a few people playing 2s. Honest expectation would be: more people playing 2s than they do right now, but less than on the s1 start.

Underlying topics, which imo should be addressed
- the rating a team starts at and the arena point calculation applied. Most people play 5s only to get arena points on the least resistance way. given that you need already a plethora of games to reach a rating in 2s to break even with the minimum points in 5s, especially if you play a class which isnt benefiting from the current imbalances and bugs. Potentially, also setting the starting rating above the even point which you would have if you have a 50:50 win-lose ratio (alas 1500) to have a higher influx of rating into the system to compensate for the smaller playerbase (if teams start at 1600, you would have an incentive to create a new time if you lose more than you win to receive more arena points every week). But, this also heavily depends on the arena point calculation which is applied. some of them scale from 0, some only start scaling from ~1500 and higher
- a discussion about disabling abilities and talents which are clearly and confirmed not working as intended, creating a clear advantage and incentive to be abused. This is different from a discussion about potentially tuning of abilities which happen to have a default wotlk scaling which just happen to be unreasonable in the progression state of chromiecraft.


Further, to prevent distribution players even more, i would suggest be more closely to retail in the sense of disabling arena for e.g. 2 weeks ("concluding a season") but enabling the honor gear of the next season. this would channel people into battlegrounds and not split people between arena and bgs (which would happen if they are released at the same point in time).

**Comment by Eledor-collab** [comment]
I'm a huge PvP enjoyer in the server, but sadly I do a very very limited amount of arenas like most of my friends and guildmates. So, I might be wrong... but this is my take away about doing arenas in Chromiecraft:

1.  **Why is 5v5s not happening anymore?** Waiting 10 mins to be farmed by the same unbeatable team over and over again is not fun.

- What my friends say about 5s? "We have ended up with the same 1 or 2 teams at 2k rating that are just gonna steamroll at you". 
- Why only a handful of people is having this much fun while the rest of us, mere mortals, gets to be farmed like veggies? Imbalance in some classes due to talents, weird scaling on certain spells/abilities (I'm looking at you, wolves!), small population, low resilience available, or simply the fact that certain people is just better at PvP (if you think this is true, just go to Hardling, one of the scariest/proficient hunters in PvP, and ask him how so he's not doing arenas at all)? 
- My answer to all of this... I don't know dude. I think it's just simply too hard to ping point which could be the "real" problem here. But 5v5s might stay as they're from now and on.

**2. Why people is doing 3v3s?** Well... points! ofc

- As most of you have already mentioned it, people were into 5v5s because of the high amount of points you get. But because apparently getting wrecked by the same number-one-team over and over again is not fun, imagine having to wait 10 min for each encounter just to get farmed again. Very nice, huh...
- 3v3s became an alternative to 5v5s because most of my friends want to avoid the situation mentioned above.
- setting up a team for 3v3s is easier! because.. well... you need only 3 dudes, instead of 5.

**3. Ok ok, maybe... maybe you're a bit right Eledor... so what?** I think releasing the next arena season with the same current conditions would just... well.... take us where we're right now. Same old rascals on top, while the rest has to find ways to have some fun doing arenas with friends to get points while not getting farmed by la crème de la crème of PvP.

- as some of you have pointed out, I also think the extra amount of resilience we'll get can't solve the problems we're facing in arenas. I might be wrong, though, but... I don't think I'm wrong haha
- I think having the new arena set available sooner could incentivize many people to do more arenas. So, I agree we'll have an spike in arenas as you mentioned. But because of how my friends feel about doing arenas in Chromie, the new gear might not be enough incentive for casual PvP enjoyers, and arenas will die again. People like new gear, absolutely yes, but again, having to fight the wombo-combo of warriors spinning their way to victory with the help of wolves that have insane healing capabilities and more HP that my poor mage, this is not fun dude.


I think we have a bunch of issues to fix before the next arena season release if we want to have a more not-so-sweaty friendly environment.

Instead of just anticipating the arena season 2, I'd propose getting creative to find ways to allow casual PvP players enjoy arenas. For instance, what about having a random queueing system? You like to play your wombo-combo? well, apparently that can create an unfair situation or certain classes. So, let's queue with random people! (Hey! this is a suggestion, this could be a terrible idea as it could be the best one I had so far, haha, calm down, it's just an example!)

Regardless, thanks for reading, and stay safe! Hope we can find a way to create a sustainable PvP environment in this nice server.

**Reply by balleny** [reply]
i think you are mixing a few things here tbh.

good players will stay on top and keep farming groups. Arena teams will climb, and at some point be out of the MMR to be instantly matched against casuals.
The only issue here is that compared to a few months back, people stopped playing even 5s because the arena points earned are a dire outlook on how long it takes to get gear as a "casual" doing 10 games a week. it makes a huge difference in motivation if you need 5-6 weeks per piece, or if you can get a piece every 2-3 weeks by playing casually - and i am not talking about the weapons.
that is just demotivating and also resulted in people not continuing to play 5s - besides the fact that it is more enjoyable for the "casual" player than the smaller brackets.

playing the same teams all the time is obviously not an incentive either. given that we had sometimes like 6-8 5s teams playing in parallel, you did face and play different teams. Overall, small server size just breaks pvp.

I am curious but cant predict it yet either, if a implementation of a solo arena queue, with individual rating - which would also be used for point distribution, in a 2s or 3s setup - throwing groups together randomly, but in a certain setup (e.g. physical damage, magic damage, healer) - randomly or while also trying to balance MMRs - could bring a long on holding change

(casual player in the sense of someone not too much interested into arena but still trying to get the arena set nevertheless)
